gast_to_code.py

```diff
@@ -195,6 +195,11 @@ def gast_router(gast, out_lang):
         return binOp_helper(gast, out_lang)
     elif gast["type"] == "boolOp":
         return out["boolOp"][out_lang](gast)
+    elif gast["type"] == "error":
+        if gast["value"] == "unsupported":
+            # Error string
+            return "Feature not supported"
```

Comment:
Can we make these error messages more explicit. Specifically it would be good to differentiate between the two messages.

"A language feature in your source code is unsupported by our translation service."
vs
"The source code fails to compile - please check your syntax."